towhee/types/image_utils.py

```diff
@@ -32,3 +33,35 @@ def to_image_color(img: Image, target_mode: str):
         raise ValueError('Can not convert image from %s to %s.' % (img.mode, target_mode))
 
     return Image(cv2.cvtColor(img, flag), target_mode.upper())
+
+
+def from_pil(pil_img):
+    '''
+    Convert a PIL.Image.Image into towhee.types.Image.
+
+    Args:
+        pil_img (`PIL.Image.Image`):
+            A PIL image.
+
+    Returns:
+        (`towhee.types.Image`)
+            The image wrapepd as towhee Image.
+    '''
+    return Image(np.array(pil_img), pil_img.mode)
+
+
+def to_pil(img: Image):
+    """
```

Comment:
this function should also support ndarray.

Reply:
should we check if image can be converted or should we allow crash at runtime?